common/src/main/scala/com/intuit/data/simplan/common/github/GithubInteractionHandler.scala

```diff
@@ -126,6 +127,18 @@ class GithubInteractionHandler(
     mergePR(owner, repoName, commitMessage, pullRequest.get)
   }
 
+  // branchName and filePath inputs can be set to null if not desired
+  def getRepoCommits(owner: String, repositoryName: String, branchName: String, filePath: String, pageSize: Int, pageNumber: Int) = {
+    val repositoryId = new RepositoryId(owner, repositoryName)
+    try {
+      println(s"Fetching github commits on branch $branchName for repository $repositoryName belonging to $owner (page $pageNumber with size of $pageSize)")
```

Comment:
Please use a logger instead of println

Reply:
removed the log since it isn't that needed